README.md

```diff
@@ -30,26 +30,44 @@ For example:
 ./target/release/server --abox examples/abox.ttl --tbox examples/rules.n3 
 ```
 
-## Roxi JS lib
-Make sure to have `wasm-pack`, `cargo-generate` and `npm` installed. Instruction can be found [here](https://rustwasm.github.io/book/game-of-life/setup.html).
-How to build RoXi through web assembly:
+## Using Roxi with Javascript/Typescript
+
+Make sure that you have `wasm-pack`, `cargo-generate` and `npm` installed. Instructions to install those can be found [here](https://rustwasm.github.io/book/game-of-life/setup.html).
+
+You can use roxi both inside a browser as well as a Node JS module.
+
+### Using Roxi inside a browser.
+
 ```
 cd js
-wasm-pack build
+wasm-pack buid
 ```
-This will generate a `pkg` folder. Now you can add RoXi as a dependency in your npm project:
+
+A `pkg` folder will be generated which you can use inside the browser. You can install the package inside your application with `npm install --save-dev /path/to/roxi/js/pkg` More information and a tutorial regarding using Webassembly within webpages using webpack can be found [here](https://rustwasm.github.io/book/game-of-life/hello-world.html#putting-it-into-a-web-page)
```

Comment:
`A pkg folder will be generated which you can use inside the browser.` 

This is confusing. It is not the folder that can be used inside the browser. Can you rephrase it to something like: `A pkg folder will be generated which contains the generated web assembly modules.`